### ISSUE TITLE
Incorrect generated param name

### ISSUE THREAD

**Issue by mzdm** [epam-cross-platform-lab/swagger-dart-code-generator#168]
Swagger/Json file example: https://cloud.ibm.com/apidocs/visual-recognition/visual-recognition-v3.json

This has parameter `AcceptLanguage` which has a name with a dash `Accept-Language`
```json
"components": {
    "parameters": {
      "AcceptLanguage": {
        "name": "Accept-Language",
        "in": "header",
```
This generates:
![image](https://user-images.githubusercontent.com/67197047/119004661-0a590480-b98f-11eb-8108-737375260e5c.png)

instead of `enums.$V3ClassifyPostAcceptLanguageMap[Accept - Language]` it should call `enums.$V3ClassifyPostAcceptLanguageMap[acceptLanguage]`

Thanks for this lib btw. 😍. Saves a lot of time.

<details>
<summary>deps</summary>

```
dependencies:
  flutter:
    sdk: flutter
  chopper: ^4.0.0

dev_dependencies:
  flutter_test:
    sdk: flutter
  build_runner: ^1.12.2
  json_annotation: 4.0.1
  json_serializable: 4.1.2
  swagger_dart_code_generator: ^2.0.0-nullsafety.7+2
  chopper_generator: ^4.0.0-nullsafety.1
```

</details>

**Comment by Vovanella95** [comment]
Hi @mzdm ! Looks like it's easy fix. We will fix it ASAP. (possible today)

**Comment by Vovanella95** [comment]
Please check 2.0.0-nullsafety.8

**Comment by Vovanella95** [comment]
@mzdm  Also thanks for examples of swagger files. It's really helpful.

**Comment by mzdm** [comment]
> Please check 2.0.0-nullsafety.8

Thanks for the fast fix! 💙 I'll try later tomorrow.

**Comment by mzdm** [comment]
Can confirm it was fixed with `2.0.0-nullsafety.8`, thanks!
